app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,17 @@
+class SessionsController < ApplicationController
+  def new
+
+  end
```

Comment:
Can we delete this method since it's empty? It will still render the view without throwing any errors